lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -381,7 +391,15 @@ export async function getSessionData(recipeImplementation: RecipeImplementation,
         sessionHandle,
     });
     if (response.status === "OK") {
-        return response.userDataInDatabase;
```

Comment:
- You should first check for version, and then check call the getSession or getSessionInformation API.. 
- Please see how other functions in this file gets the `querier` -> `await recipeImplementation.querier`

Reply:
Makes sense, done